src/components/Header/index.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useRef, useState } from "react";
+import { data } from "../../data/test";
+import Box from "./Boxinput";
+import BoxinputForcus from "./BoxinputForcus";
+import "./style.scss";
+import {
+  faArrowLeft,
+  faArrowRight,
+  faArrowUp,
+  faCancel,
+  faDiamond,
+  faDownload,
+  faGroupArrowsRotate,
+  faIndustry,
+  faSearch,
+  faTShirt,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+Headers.propTypes = {};
+
+function Headers(props) {
+  const [text, setText] = useState("");
+  const [box, setBox] = useState(false);
+  const inputUseRef = useRef(null);
+
+  useEffect(() => {
+    if (box) {
+      inputUseRef.current.focus();
+    }
+  }, [box]);
+  const handleCancel = () => {
+    try {
+      setText("");
+    } catch (error) {
+      console.log("dlfsf", error);
+    }
+  };
+  const filterData = (text) => {
+    const arrFilter = data.filter((item) => item.Title.includes(text));
+    return arrFilter;
+  };
+  const handleText = (e) => {
+    setText(e.target.value);
+  };
+  const handleBox = (e) => {
+    if (
+      e.target.className === "Headers__input--box" ||
+      e.target.className === "Headers__inputForcus--posi" ||
+      e.target.className === "Headers__list--title"
+    ) {
+      setBox(true);
+      return;
+    } else {
+      setBox(false);
+    }
+    console.log(e.target);
+  };
+
+  return (
+    <div
+      className="Headers"
+      onClick={(e) => {
+        handleBox(e);
+      }}
+    >
+      <div className="Headers__left">
+        {" "}
+        <div className="Headers__translate">
+          <button className="Headers__translate--left">
+            {" "}
+            <FontAwesomeIcon icon={faArrowLeft} />
+          </button>
+          <button className="Headers__translate--right">
+            {" "}
+            <FontAwesomeIcon icon={faArrowRight} />
+          </button>
+        </div>
+        {/* imput */}
+        {box === false ? (
+          <Box
+            handleText={handleText}
+            handleBox={handleBox}
+            handleCancel={handleCancel}
+            faCancel={
+              <FontAwesomeIcon icon={faCancel} onClick={handleCancel} />
+            }
+            faSearch={<FontAwesomeIcon icon={faSearch} />}
+            text={text}
+          />
+        ) : (
+          <BoxinputForcus
```

Comment:
Sai chính tả và cách đặt tên componnet em nhé, lần sau nhớ check translate trước 
```suggestion
<BoxInputFocus
```